<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imgToSend">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setImgBtn">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>189</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>select image</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textToSend">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sndBtn">
    <property name="geometry">
     <rect>
      <x>544</x>
      <y>390</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>291</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startServer">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Server</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>sendbutton()</slot>
  <slot>startButton()</slot>
 </slots>
</ui>
